server/views/tagManager/headContent.html

```diff
@@ -7,17 +7,16 @@
   ;(function (w, d, s, l, i) {
     w[l] = w[l] || []
     w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' })
-    var f = d.getElementsByTagName(s)[0],
+    const f = d.getElementsByTagName(s)[0],
```

Comment:
Checking analytics it looks like 7.15% of users use IE 11.

There is some support for `const` and this should be alright, just need to be wary that `const` is not supported in for-in and for-of loops